src/features/import/components/Configure/Preview/index.tsx

```diff
@@ -0,0 +1,174 @@
+import { ArrowBackIos, ArrowForwardIos } from '@mui/icons-material';
+import { Box, Button, Typography, useTheme } from '@mui/material';
+import { useEffect, useState } from 'react';
+
+import FieldsPreview from './FieldsPreview';
+import messageIds from 'features/import/l10n/messageIds';
+import { Msg } from 'core/i18n';
+import OrgPreview from './OrgPreview';
+import PreviewGrid from './PreviewGrid';
+import TagsPreview from './TagsPreview';
+import { useNumericRouteParams } from 'core/hooks';
+import usePersonPreview from 'features/import/hooks/usePersonPreview';
+import useSheets from 'features/import/hooks/useSheets';
+import { ColumnKind, Sheet } from 'features/import/utils/types';
+
+const Preview = () => {
+  const theme = useTheme();
+  const { sheets, selectedSheetIndex, firstRowIsHeaders } = useSheets();
+  const [personIndex, setPersonIndex] = useState(0);
+  const currentSheet: Sheet = sheets[selectedSheetIndex];
+  const { orgId } = useNumericRouteParams();
+  const { fields, org, tags } = usePersonPreview(
+    currentSheet,
+    firstRowIsHeaders ? personIndex + 1 : personIndex,
+    orgId
+  );
+
+  const emptyPreview = currentSheet.columns.every(
+    (item) => item.selected === false
+  );
+
+  const tagColumnSelected = currentSheet.columns.some(
+    (column) => column.kind === ColumnKind.TAG && column.selected
+  );
+
+  useEffect(() => {
+    setPersonIndex(0);
+  }, [selectedSheetIndex]);
+
+  return (
+    <Box p={2}>
+      <Box alignItems="center" display="flex" sx={{ mb: 1.5 }}>
+        <Typography sx={{ mr: 2 }} variant="h5">
+          <Msg id={messageIds.configuration.preview.title} />
+        </Typography>
+        <Button
+          disabled={personIndex === 0}
+          onClick={() =>
+            setPersonIndex((prev) => (personIndex !== 0 ? prev - 1 : prev))
```

Comment:
On this line, I would expect `personIndex` to be `prev`, because otherwise you're not getting the full benefit of using a setter callback instead of just setting the value directly.